src/constants/v3Farms.ts

```diff
@@ -23,10 +38,40 @@ export const DpiEthRewards: V3Farm = {
       startTime: 1629493200,
       endTime: 1630789200,
       refundee: '0x9467cfADC9DE245010dF95Ec6a585A506A8ad5FC',
+      dateText: 'August 20th, 2021 - September 4th, 2021',
+      img: {
+        alt: 'DPI Icon',
+        src: 'https://set-core.s3.amazonaws.com/img/social_trader_set_icons/defi_pulse_index_set.svg',
+      },
     },
   ],
 }
 
-const farms = [DpiEthRewards]
+export const MviEthRewards: V3Farm = {
+  tokenPair: 'MVI/ETH',
+  pool: '0x9359c87B38DD25192c5f2b07b351ac91C90E6ca7',
+  poolLabel: 'MVI-ETH',
+  img: {
+    alt: 'MVI Icon',
+    src: 'https://set-core.s3.amazonaws.com/img/social_trader_set_icons/defi_pulse_index_set.svg',
+  },
+  farms: [
+    // August 20th - September 4th
+    {
+      rewardToken: '0x0954906da0Bf32d5479e25f46056d22f08464cab', //INDEX
```

Comment:
[question] do we want to pull this token address from environment variables? 
```suggestion
      rewardToken: process.env.REACT_APP_INDEX_TOKEN_ADDRESS as string, // INDEX
```

Reply:
I wouldn't bother, as this prevents it from getting crossed up if the env var changes accidentally